pkgs/tools/cd-dvd/unetbootin/default.nix

```diff
@@ -12,7 +12,9 @@ stdenv.mkDerivation rec {
     sha256 = "18bbcrjk6ladr46kl3dvqz5pq2xcv4nnwmajqllb4sl3k1xqsngy";
   };
 
-  sourceRoot = "source/src/unetbootin";
+  setSourceRoot = ''
+    sourceRoot=$(echo */src/unetbootin)
+  '';
 
```

Comment:
It's been suggested not to use "source" directly - https://github.com/NixOS/nixpkgs/pull/31106#issuecomment-341266441 - hence I changed it as per the comment.